docker_cli/cli/build_commands.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+from typing import List
+
+from ..commands import clone
+from ._utils import add_tag_argument, help_formatter
+
+
+def init_build_parsers(subparsers: argparse._SubParsersAction) -> None:
+    """
+    Augment an argument parser with build commands.
+
+    Parameters
+    -------
+    parser : argparse.ArgumentParser
+        The parser to add setup commands to.
+    prefix : str
+        The image tag prefix.
+    """
+    clone_params = argparse.ArgumentParser(add_help=False)
+    clone_params.add_argument(
+        "--repo",
+        type=str,
+        metavar="GIT_REPO",
+        default="isce-framework/isce3",
```

Comment:
Given our changes to the lower-level function, let's remove this default value and make this parameter required

```suggestion
```